src/main/java/ua/java/caesar/caesarcypher/CaesarCypher.java

```diff
@@ -0,0 +1,59 @@
+package ua.java.caesar.caesarcypher;
+
+import ua.java.caesar.alphabets.Alphabet;
+
+import java.util.List;
+
+
+public class CaesarCypher {
+    final List<Character> ALPHABET;
```

Comment:
1) why not private access modifier?
2) why do we use here naming the same as for constants (static final)?

Reply:
please fix the problem of final naming in all your code